homeassistant/components/climate/sensibo.py

```diff
@@ -29,7 +29,7 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-ALL = 'all'
+ALL = ['all']
```

Comment:
This shouldn't be necessary since voluptuous will pass `ALL` through the validator, which will be turned into a list by `cv.ensure_list`.

Reply:
![image](https://user-images.githubusercontent.com/1444314/36915004-f7d177b8-1e03-11e8-8466-0051305de163.png)

Reply:
The part that fails is comparing the result (['all']) with `ALL` on line 73.